main_test.go

```diff
@@ -285,7 +285,12 @@ func TestStartMain(t *testing.T) {
 
 	os.Args = []string{"test-autograph", "-D"}
 	go main()
-	time.Sleep(200 * time.Millisecond)
+	if os.Getenv("CI") == "true" {
```

Comment:
maybe `!= ""` ?

Reply:
Going to leave as is and we can change them both in another PR. Does seem weird to depend on CI setting this specific value.